packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/translations/Messages_fr_FR.json

```
@@ -29,6 +29,7 @@
   "nutanix_redeploy_custom_config_ip_unique_error": "Cette IP est déjà utilisée sur ce cluster",
   "nutanix_redeploy_custom_config_ip_not_in_cidr_subnet_error": "Les adresses ne sont pas sur le même réseau.",
   "nutanix_redeploy_custom_config_ip_format_error": "Le format de l'IP n'est pas correct",
+  "nutanix_redeploy_custom_config_vlan_id_error": "Le champ VLAN ID doit être un numéro compris entre 0 et 4000",
```

Sounds a bit weird to me, can you check the translation with PO
```suggestion
  "nutanix_redeploy_custom_config_vlan_id_error": "Le champ VLAN ID doit être un nombre compris entre 0 et 4000",
```